www/install/sql/centreon/Update-DB-20.10.0.sql

```diff
@@ -4,6 +4,7 @@ CREATE TABLE IF NOT EXISTS `user_filter` (
     `user_id` int(11) NOT NULL,
     `page_name` varchar(255) NOT NULL,
     `criterias` text,
+    `order` int(11) NOT NULL,
```

Comment:
If order cannot be NULL : 
What will happen for platforms migrating from 20.04.5 with filters ?

Reply:
this is a "CREATE TABLE", not "UPDATE" ;)